base/sparse/cholmod.jl

```diff
@@ -87,22 +87,22 @@ function __init__()
             tmp = Array(Cint, 3)
             ccall((:cholmod_version, :libcholmod), Cint, (Ptr{Cint},), version_array)
```

Comment:
`version_array` ends up with the linked version of CHOLMOD if possible and `tmp` has the version at build time. Therefore, I think we could just have
```jl
tmp = copy(version_array)
```
instead of 
```jl
tmp = Array(Cint, 3)
ccall((:jl_cholmod_version, :libsuitesparse_wrapper), Cint, (Ptr{Cint},), tmp)
```
but I think there is a problem with `version` which should be updated at run time.

I think we need these definitions at compile time, but I'm not completely sure, so it might be that they can just be defined in the ``__init__`` function.